src/Cli/dotnet/Commands/Test/MTP/SolutionAndProjectUtility.cs

```diff
@@ -338,7 +338,7 @@ static RunProperties GetRunProperties(ProjectInstance project)
         }
     }
 
-    private static ProjectLaunchSettingsModel? TryGetLaunchProfileSettings(string projectDirectory, string projectNameWithoutExtension, string appDesignerFolder, BuildOptions buildOptions, string? profileName)
+    private static LaunchSettingsModel? TryGetLaunchProfileSettings(string projectDirectory, string projectNameWithoutExtension, string appDesignerFolder, BuildOptions buildOptions, string? profileName)
```

Comment:
@dotnet/dotnet-testing if we expand the 'launch settings' handlers that are known, how should the testing experience react? does `test` support launch settings profiles today?